test-interface/src/main/scala/org/scalajs/testinterface/internal/FrameworkDetector.scala

```diff
@@ -0,0 +1,37 @@
+package org.scalajs.testinterface.internal
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+import scala.scalajs.js.JSConverters._
+import scala.scalajs.reflect.Reflect
+
+private[internal] object FrameworkDetector {
+  @JSExportTopLevel("org.scalajs.testinterface.internal.detectFrameworks")
+  def detectFrameworks(exportsNamespace: js.Dynamic,
```

Comment:
Indeed. I blindly ported the parameter from the JS code in `FrameworkDetector`. We definitely can use `envInfo.exportsNamespace` instead :)